src/elements/kw-app/kw-app.html

```diff
@@ -215,7 +216,7 @@
         <kw-auth-modal id="auth-modal"></kw-auth-modal>
     </template>
     <script>
-        const catchUrls = [/^\/projects$/, /^\/welcome$/];
+        const catchUrls = [/^\/projects$/, /^\/welcome$/,/^\/shares/,/^\/element/];
```

Comment:
When you make a PR, make sure you remove this. Otherwise it will break redirections in production